lib/BoundaryClickWatcher/index.js

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
In this instance I can live with it a click handler on a div. Otherwise what's the alternative?